### DISCUSSION TITLE
Some translation in Chinese (traditional) [zh-TW]

### DISCUSSION THREAD

**Discussion by sjwang1100** [aclap-dev/video-downloadhelper#945] [Language & Translation]
I saw a few strings missing on Chinese (traditional) locale, and decided to try to contribute some. Here's my result

[zh_TW.txt](https://github.com/user-attachments/files/16023880/zh_TW.txt)

Note.
Firefox translates "addon" to "附加元件", so I thought I'd align with them instead of using "外掛程式" that I heard more often.

Please let me know if you have any comments or improvement.

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
